test/evergreen.yml

```diff
@@ -173,6 +173,28 @@ functions:
               -G "${cmake_generator|Ninja}" ../.
           fi
     - *make_wiredtiger
+  # FIXME-WT-8981 - Ubuntu PPC tests have been replaced by RHEL8 tests as of WT-8688, but the RHEL8 tests are currently disabled 
+  # due to platform issues. On completion of WT-8981 this can be deleted.
+  "compile wiredtiger address sanitizer ppc ubuntu":
+    - command: shell.exec
+      params:
+        working_dir: "wiredtiger"
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
+          if [ "$OS" != "Windows_NT" ]; then
+            . test/evergreen/find_cmake.sh
+            if [ -d cmake_build ]; then rm -r cmake_build; fi
+            mkdir -p cmake_build
+            cd cmake_build
+            $CMAKE -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v3_clang.cmake -DCMAKE_C_FLAGS="-ggdb" -DWITH_PIC=1 \
+              -DHAVE_DIAGNOSTIC=1 ${NON_BARRIER_DIAGNOSTIC_YIELDS|} -DCMAKE_BUILD_TYPE=ASan \
+              -DHAVE_BUILTIN_EXTENSION_LZ4=1 -DHAVE_BUILTIN_EXTENSION_SNAPPY=1 -DHAVE_BUILTIN_EXTENSION_ZLIB=1 ${configure_python_setting|} \
+              -G "${cmake_generator|Ninja}" ../.
+          fifo
```

Comment:
autocomplete fail? i think this should be `fi`.

Reply:
Yes it should be, thanks!. Not sure how that passed my patch tests..

Reply:
Fixed in 6ebcabca06689c5485eabb04a16165e5693e7ecc